upgrade/1.0/scripts/ceph/lib/ceph-service-status.sh

```diff
@@ -259,7 +258,7 @@ then
      then
        for osd in $(ceph orch ps --daemon_type osd --hostname "$host" -f json-pretty |jq -r '.[]|(.daemon_type+"."+.daemon_id)')
        do
-         check_service
+         check_service 
```

Comment:
I suppose this came in on a commit to staging/1.0.1 earlier, but there's an extra space here.